terraform/container-definitions/web_container_definition.json

```diff
@@ -126,6 +126,10 @@
         "name": "AUTH_SPREADSHEET_ID",
         "value": "${auth_spreadsheet_id}"
       },
+      {
+        "name": "PUBLISHED_VACANCY_SPREADSHEET_ID",
+        "value": "${published_vacancy_spreadsheet_id}"
+      },
```

Comment:
This needs to be configured in the `worker_container_definition` as that is where the jobs are executed.